src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.IncrementalMemberEditAnalyzer.cs

```diff
@@ -179,20 +177,6 @@ async Task ExecuteAnalyzersAsync(
                     foreach (var analyzerWithState in analyzersWithState)
                     {
                         var diagnostics = await computeAnalyzerDiagnosticsAsync(analyzerWithState.Analyzer, executor, cancellationToken).ConfigureAwait(false);
-
-                        // If we computed the diagnostics just for a span, then we are performing incremental analysis.
-                        // We need to compute the full document diagnostics by re-using diagnostics outside the changed
-                        // member and using the above computed latest diagnostics for the edited member span.
-                        if (analysisScope.Span.HasValue)
-                        {
-                            Debug.Assert(analysisScope.Span.Value == changedMember.FullSpan);
-
-                            diagnostics = await GetUpdatedDiagnosticsForMemberEditAsync(
-                                diagnostics, analyzerWithState.ExistingData, analyzerWithState.Analyzer,
-                                executor, changedMember, changedMemberId,
-                                oldMemberSpans, computeAnalyzerDiagnosticsAsync, cancellationToken).ConfigureAwait(false);
```

Comment:
analyzeWithState.ExistingData would always be empty.  as such, this method could not actually update anything.